format/Schema.fbs

```diff
@@ -107,9 +107,19 @@ table FloatingPoint {
 table Utf8 {
 }
 
+/// Opaque binary data
 table Binary {
 }
 
+/// Same as Utf8 and Binary, but with 64-bit offsets, allowing to represent
+/// extremely large data values.
+/// Those two types are optional and not necessarily available in all implementations.
```

Comment:
I thought the consensus that was this this shouldn't be the case for new types, we should do our best to at least partially support them in Java (even if they can't be fully supported well).

Reply:
I don't know. What do you suggest? Simply remove this line?

Reply:
I've removed it in any case.